poglink/cogs/rates.py

```diff
@@ -19,43 +19,26 @@
 
 # create cog class
 class Rates(commands.Cog):
-    DEFAULT_SERVER_INFO = {
-        "smalltribes": {
-            "short_name": "Smalltribes",
-            "color": 0xA34C44,
-        },
-        "arkpocalypse": {
-            "short_name": "Arkpocalypse",
-            "color": 0xF8DE74,
-        },
-        "conquest": {
-            "short_name": "Conquest/Classic",
-            "color": 0x6DFF90,
-        },
-        None: {
-            "short_name": "Official",
-            "color": 0x63BCC3,
-        },
-    }
```

Comment:
Removed lookup dict, no longer needed, explicitly providing this in `config.yaml` now

Reply:
hm, without this set of defaults, where do things fall back to without the config file? I suspect that's related to the current test failures.

I also installed and ran locally, and though I'm seeing similar test failures, I don't see all of them. Only 2 tests fail due to an empty dict key.

But I did notice a lot of dependency conflicts that might be causing some discrepancies between local and CI. I think that also would be worth ironing out and pinning down more strictly. Most of our `setup_requires` list has no restrictions and a lot of the packages I have introduced breaking/incompatible changes.

Reply:
Ah wait, you didn't update the test data config files to match the new structure you're expecting. That might be it.

Reply:
Ye thats whats doing on, the defaults are still defined in config.yaml, this was just named poorly